docs/using-ngrok-with/docker.md

```diff
@@ -39,6 +39,12 @@ endpoints:
       url: http://host.docker.internal:80
 ```
 
+If you are unable to view the web inspection interface typically available at `http://localhost:4040`, you may need to map your host port `4040` to port `4040` on the container, for example:
+
+```bash
+docker run -p 4040:4040 -it -e NGROK_AUTHTOKEN=xyz ngrok/ngrok:latest http 80
```

Comment:
I think we should retain usage of the special `host.docker.internal` URL here. The combination of exposing the port explicitly and using `host.docker.internal` is what worked for the user who [kicked off the discussion](https://github.com/ngrok/ngrok/issues/90).

```suggestion
docker run -p 4040:4040 -it -e NGROK_AUTHTOKEN=xyz ngrok/ngrok:latest http host.docker.internal:80
```